packages/edit-site/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -2,11 +2,22 @@
  * WordPress dependencies
  */
 import { useSelect } from '@wordpress/data';
+import { useEntityProp } from '@wordpress/core-data';
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { wordpress } from '@wordpress/icons';
 
 function FullscreenModeClose( { icon } ) {
+	const [ siteIconURL ] = useEntityProp( 'root', 'site', 'site_icon_url' );
+	
+	const isRequestingSiteIcon = useSelect( ( select ) => {
+		return ! select( 'core/data' ).hasFinishedResolution(
```

Comment:
I normally use `isResolving` which I _think_ is just `! hasFinishedResolution` (haven't checked tho).

```suggestion
		return select( 'core/data' ).isResolving(
```